### DISCUSSION TITLE
Libre Computer Alta AML-A311D-CC

### DISCUSSION THREAD

**Discussion by sallyhaj** [johang/sd-card-images#128] [General]
Hello all,
I recently got Alta AML-A311D-CC board from Libre Computer, and as I heard, it should be working with a general EFI aarch64 image.
Is there any plan to add this broad to the list of supported device?
Is there any alternative/general image, which might work with this board?
Thank you.

**Comment by johang** [comment]
I don't think this board is supported by mainline U-Boot. One it's supported I will probably add it

**Comment by Juffo-Wup** [comment]
The Libre Computer Alta AML-A311D-CC now appears to be supported by the latest version of U-Boot.  See:
https://docs.u-boot.org/en/latest/board/amlogic/aml-a311d-cc.html

Thank you for your SD card images tool!

**Comment by butonic** [comment]
booting debian-trixie-arm64-ang7iy.bin.gz ethernet is not enabled ootb. To enable it run `networkctl up end0`.
